5a/run-client-local.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-export ADVERTISEDIP=`ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v -m1 '127.0.0.1'`
+export ADVERTISEDIP=$(ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v -m1 '127.0.0.1')
```

Comment:
If you're changing this script (and also the `run-client-docker.sh`, this will have to apply yo all other solutions as well. Please change it there as well (maybe in a separate PR).

Reply:
All the `run-client-*.sh` are working perfectly per the screenshot I added to the PR.